pages/docs/environments/gitpod.mdx

```diff
@@ -2,13 +2,67 @@ import DocsLayout from "@components/layouts/DocsLayout.js";
 
 ## Gitpod
 
-In the cloud free development environment which will have all the dependencies you need (for example MongoDB).
+Gitpod is an open-source application that provides a prebuilt development environment in your browser - powered by VS Code. It also automates the provisioning of a ready-to-code developer environment, so you will have all the dependencies you need (for example MongoDB). With Gitpod, there’s no need to download any code to your machine. It can create a server-side-dev-environment for you in seconds from any repository on GitHub.
+
+### How to use Gitpod
 
 You can use Gitpod in the cloud. Please note, that Gitpod requires additional configuration to work on the authenticated pages in LinkFree.  See instructions for [how to set up your OAuth app](/docs/environments/environment-variables) for more information.
 
-Setup environment [variables](/docs/environments/environment-variables)
+There are 2 ways you can open a repository with Gitpod:
+
+- Prefix the url with `gitpod.io/#`
+- Use the [Browser Extension](https://www.gitpod.io/docs/configure/user-settings/browser-extension)
+
+### Development
+
+Once you have opened a repository, you can develop as you usually do! In the next example, we will update our profile from LinkFree.
+
+**_NOTE:_** Be sure that you have a fork of LinkFree in your Github Profile. If you don't have it, check out the [Quickstart Guide](https://linkfree.io/docs/quickstart).
+
+Open your fork and follow along.
+
+First, Let's make sure that we have the latest version of the code. To achieve this, run the following commands:
+
+```bash
+git checkout main
+git pull upstream main
+```
+
+Now we can update our profile. To do this, create a new branch with the following command
+
+```bash
+# Replace YOUR_PROFILE_NAME with your actual profile name.
+git checkout -b update-YOUR_PROFILE_NAME
+```
+
+Now we are ready to make some modification to our profile. Find the file with your account name in the `data` folder, and modify it as you want.
+
+For example, you can copy and past the following code
+
+```json
+"milestones": [
+    {
+      "title": "Used Gitpod to Update my Profile",
+      "date": "May 2023", // update to the correct date
+      "icon": "FaCode",
+      "color": "#FFAE33",
+      "description": "Started my profile by using Gitpod",
+      "url": "https://gitpod.com/"
+    }
+  ]
+```
+
+As a last step, commit the changes and create a PR in the LinkFree Repository.
+
+### Permissions
+
+If you encounter permission issues, follow the next steps:
+
+- Open the user settings of your Gitpod profile and click on _Integrations_.<br />
+- Click on the 3 dots after Github and choose _Edit Permissions_.<br />
+- Make sure that you have checked _public_repo_ and _workflow_
```

Comment:
As these are bullet points, they will be on separate lines and do not need `<br />`

```suggestion
- Open the user settings of your Gitpod profile and click on _Integrations_
- Click on the 3 dots after Github and choose _Edit Permissions_
- Make sure that you have checked _public_repo_ and _workflow_
```